srcpkgs/godot/template

```diff
@@ -42,15 +40,29 @@ pre_build() {
 	export CXXFLAGS=" $CXXFLAGS "
 }
 
+do_build() {
+	scons ${makejobs} ${make_build_args} CCFLAGS="$CFLAGS" LINKFLAGS="$LDFLAGS" tools=yes target=release_debug
+	scons ${makejobs} ${make_build_args} CCFLAGS="$CFLAGS" LINKFLAGS="$LDFLAGS" tools=no target=release_debug
+	scons ${makejobs} ${make_build_args} CCFLAGS="$CFLAGS" LINKFLAGS="$LDFLAGS" tools=no target=release_debug platform=server
+	scons ${makejobs} ${make_build_args} CCFLAGS="$CFLAGS" LINKFLAGS="$LDFLAGS" tools=no target=release
+	scons ${makejobs} ${make_build_args} CCFLAGS="$CFLAGS" LINKFLAGS="$LDFLAGS" tools=no target=release platform=server
```

Comment:
Please try making a loop for these so the command isn't repeated that many times.

Reply:
This is adding `release` targets, we only had `release_debug` targets previously.

Reply:
`release_debug` is necessary when you use `tools=yes` (the editor), without this PR you only provide the editor to develop games, but with this PR you also provide possibility to actually ship games even on musl libc, for that reason it's necessary to have both `release` and `release_debug` export templates, besides you need those extra binaries when someone will start packaging games made with godot but not linked against musl libc